libraries/cyclestreets-core/src/main/java/net/cyclestreets/api/client/RetrofitApiClient.java

```diff
@@ -141,20 +141,20 @@ public String getJourneyJson(final String plan,
                                final String arriving,
                                final int speed) throws IOException {
     Response<String> response = v1Api.getJourneyJson(plan, itineraryPoints, leaving, arriving, speed).execute();
-    return JourneyStringTransformer.fromV1ApiJson(response.body());
+    return JourneyStringTransformer.INSTANCE.fromV1ApiJson(response.body());
```

Comment:
I think we can avoid this .INSTANCE. business by putting the fromV1APIJson method inside a companion object.

Reply:
Cool, I'll look into that.